### PR TITLE
Feature/boas 673 amend labels

### PR DESCRIPTION
**Pull request by Xfarya**
## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/.env.development [hunk 1]
```diff
@@ -31,7 +31,7 @@ APPEALS_CASE_OFFICER_GROUP_ID=appeals_case_officer
 APPEALS_INSPECTOR_GROUP_ID=appeals_inspector
 APPEALS_VALIDATION_OFFICER_GROUP_ID=appeals_validation_officer
 APPLICATIONS_CASE_ADMIN_OFFICER_GROUP_ID=applications_case_admin_officer
-APPLICATIONS_CASEOFFICER_GROUP_ID=applications_case_officer
+APPLICATIONS_CASETEAM_GROUP_ID=applications_case_team
```
Comment (francescobrenci-AR) [inline apps/web/.env.development:34]: All the env vars need to be updated in Azure and the infrastructure as well before this can be merged

**Review by francescobrenci-AR — CHANGES_REQUESTED** [review]

**Review by lutamKC — APPROVED** [review]

File: apps/web/.env.development (hunk 1 above)
Comment (francescobrenci-AR) [reply apps/web/.env.development:34]: env vars updated: https://portal.azure.com/#@planninginspectorate.gov.uk/resource/subscriptions/962e477c-0f3b-4372-97fc-a198a58e259e/resourceGroups/pins-rg-back-office-dev-ukw-001/providers/Microsoft.Web/sites/pins-app-back-office-wfe-dev-ukw-001/configuration

**Pull request merged**